src/lib.rs

```diff
@@ -567,6 +567,16 @@ pub use payload::*;
 pub use upload::*;
 pub use util::{coalesce_ranges, collect_bytes, GetRange, OBJECT_STORE_COALESCE_DEFAULT};
 
+/// Type alias for HTTP extensions used in object store operations
```

Comment:
What's the motivation for using a type alias Vs re-exporting the http type?

Reply:
To be honest I didn't think of it much. At the moment it felt cleaner rather than to use `::http::Extensions` everywhere. But I guess I could put it next to the other two imports and it won't change a thing.

Reply:
Will update the code and push again.

Reply:
I think I would prefer a re-export, as I think it is potentially confusing otherwise - can the types be used interchangeably or should people use object_store::Extensions in case we redefine it?